power.c

```diff
@@ -0,0 +1,21 @@
+//Emma L
+//powers below n^2
+
+
+#include <stdio.h>
+#include <math.h>
+int main(){
+	printf("squares numbers under 1000\n");
+
+	int x;
+
+	x=0;
+	int square;
```

Comment:
You should have initialized this to 0 so that your program could actually print all the powers of 2.